src/Adapter/Order/CommandHandler/ChangeOrderDeliveryAddressHandler.php

```diff
@@ -39,6 +41,19 @@
  */
 final class ChangeOrderDeliveryAddressHandler extends AbstractOrderHandler implements ChangeOrderDeliveryAddressHandlerInterface
 {
+    /**
+     * @var OrderAmountUpdater
+     */
+    private $orderAmountUpdater;
+
+    /**
+     * @param OrderAmountUpdater $orderAmountUpdater
+     */
+    public function __construct(OrderAmountUpdater $orderAmountUpdater)
```

Comment:
use `PrestaShop\PrestaShop\Adapter\Configuration`;

Reply:
I thought about it but the adapter service doesn't allow to specify the shop id..